stripe-buffer.el

```diff
@@ -136,9 +174,25 @@ Used by `stripe-table-mode' Only the first matching group will be painted."
                                     (goto-char end)
                                     (point))))))
                       ( overlay (apply get-overlay-create-function stripe-region)))
-                 (overlay-put overlay 'face stripe-highlight-face)
-                 (overlay-put overlay 'is-stripe t)
-                 (push overlay sb/overlays)))))
+
+                 (cond
+                  ((unless (require 'dired-subtree nil t)
```

Comment:
What you wrote is not equivalent to `featurep`. It's extremely slow if it the library is not installed.